api4/drafts.go

```diff
@@ -121,7 +121,14 @@ func deleteDraft(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	draft, err := c.App.GetDraft(userID, channelID, rootID)
```

Comment:
I was following the call stack for the `GetDraft` app call. I noticed a small bug in the app method:

https://github.com/mattermost/mattermost-server/blob/master/app/draft.go#L28


```
return nil, model.NewAppError("GetDraft", "app.draft.get.app_error", nil, nfErr.Error(), http.StatusNotFound)
```

should be 

```
return nil, model.NewAppError("GetDraft", "app.draft.get.app_error", nil, err.Error(), http.StatusNotFound)
```

`nfErr` -> `err`, `nfErr` is the sentinel value, it doesn't have the context that `err` has from the store layer.

Reply:
Good catch, will make the change 👍